src/wallet/init.cpp

```diff
@@ -208,7 +208,7 @@ bool VerifyWallets()
         }
 
         std::string strError;
-        if (!CWalletDB::VerifyEnvironment(walletFile, GetDataDir().string(), strError)) {
+        if (!CWalletDB::VerifyEnvironment(walletFile, GetWalletDir().string(), strError)) {
```

Comment:
The arguments in the declaration and definition for `VerifyEnvironment()` and `VerifyDatabaseFile()` are still `dataDir`. Those should be updated to indicate that they're now refering to the wallet directory.

Reply:
Fixed